packages/backend-core/src/features/index.ts

```diff
@@ -277,5 +277,7 @@ export const flags = new FlagSet({
   AUTOMATION_BRANCHING: Flag.boolean(env.isDev()),
   SQS: Flag.boolean(env.isDev()),
   [FeatureFlag.AI_CUSTOM_CONFIGS]: Flag.boolean(env.isDev()),
-  [FeatureFlag.ENRICHED_RELATIONSHIPS]: Flag.boolean(false),
+  [FeatureFlag.ENRICHED_RELATIONSHIPS]: Flag.boolean(
+    env.isDev() && !env.isTest()
```

Comment:
The tests are need to be rewritten to support the `ENRICHED_RELATIONSHIPS` by default

Reply:
I think we should probably update the test cases to do this rather than putting the `!env.isTest()` in here, that could be quite dangerous/missed or forgotten.